analysed-packages/exfat-utils/version-1.3.0/exfat-utils-1.3.0-SPDX2TV.spdx

```diff
@@ -0,0 +1,2024 @@
+Copyright 1992-2016 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: exfat-utils-1.3.0.tar.gz/exfat-utils-1.3.0.tar/exfat-utils-1.3.0/config.sub
+SPDXID: SPDXRef-item816300
+FileChecksum: SHA1: c15705cb56948877efbe9558541fbb4940e62155
+FileChecksum: SHA256: 945606bea770c8facc4d715a8d7d0c11c21a2d2214ad6ab7e859b55a6bb90ed2
+FileChecksum: MD5: 8c238eb18a69e902d34b48a1a3702c02
+LicenseConcluded: LicenseRef-GPL-3.0-or-later-WITH-Autoconf-exception-generic-3.0
+LicenseInfoInFile: LicenseRef-GPL-3.0-or-later
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic-3.0
+LicenseInfoInFile: LicenseRef-free-unknown
+LicenseInfoInFile: LicenseRef-gpl-3.0-plus
+LicenseInfoInFile: LicenseRef-autoconf-simple-exception
+FileCopyrightText: <text> Copyright 1992-2016 Free Software Foundation, Inc.
+Copyright 1992-2016 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: exfat-utils-1.3.0.tar.gz/exfat-utils-1.3.0.tar/exfat-utils-1.3.0/compile
+SPDXID: SPDXRef-item816301
+FileChecksum: SHA1: 03a05e37267c2f1c9fea4f2a6a15f8a7beb91c0d
+FileChecksum: SHA256: 41856428226c049fa0ea4ca7a574c28416de2769b55ed7feea4ab5e60283af16
+FileChecksum: MD5: 500b9244caa7a7ab23ece1db37efa76d
+LicenseConcluded: LicenseRef-GPL-2.0-or-later-WITH-Autoconf-exception-generic
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic-3.0
+LicenseInfoInFile: LicenseRef-gpl-2.0-plus
+LicenseInfoInFile: LicenseRef-autoconf-simple-exception-2.0
+FileCopyrightText: <text> Copyright (C) 1999-2014 Free Software Foundation, Inc. Written by Tom Tromey <tromey@cygnus.com>. </text>
+
+##File
+
+FileName: exfat-utils-1.3.0.tar.gz/exfat-utils-1.3.0.tar/exfat-utils-1.3.0/ar-lib
+SPDXID: SPDXRef-item816279
+FileChecksum: SHA1: bd59743e475ef3fa5ac10601b3477ee74cd5e794
+FileChecksum: SHA256: 73f97940d6551db8f0b266941e318571aac460bf3bbf364e445ebeebcfcfdd16
+FileChecksum: MD5: 1f44d8791219b3da98e50128bb6b0938
+LicenseConcluded: LicenseRef-GPL-2.0-or-later-WITH-Autoconf-exception-generic
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic-3.0
+LicenseInfoInFile: LicenseRef-gpl-2.0-plus
+LicenseInfoInFile: LicenseRef-autoconf-simple-exception-2.0
+FileCopyrightText: <text> Copyright (C) 2010-2014 Free Software Foundation, Inc. Written by Peter Rosin <peda@lysator.liu.se>. </text>
+
+##File
+
+FileName: exfat-utils-1.3.0.tar.gz/exfat-utils-1.3.0.tar/exfat-utils-1.3.0/aclocal.m4
+SPDXID: SPDXRef-item816308
+FileChecksum: SHA1: 8a87492b1a58ba261eb82e4b18fd14f292c31ee7
+FileChecksum: SHA256: e3af379bdd17553898a6edade1624f79294853ed3dcf70210c6aa6c1b562f7ed
+FileChecksum: MD5: 8c0c7c050a9cf260a19d294c98741fd7
+LicenseConcluded: LicenseRef-FSFULLR AND LicenseRef-FSFULLRWD AND LicenseRef-GPL-2.0-or-later-WITH-Autoconf-exception-generic
```

Comment:
The conclusion FSFULLRWD is correct, but can you please provide as license text the text that is given in the file:
This file is free software; the Free Software Foundation
gives unlimited permission to copy and/or distribute it,
with or without modifications, as long as this notice is preserved.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY, to the extent permitted by law; without
even the implied warranty of MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.
and not the default text which is: "This Makefile.in is free software; ..."